src/send/EnterAmount.tsx

```diff
@@ -427,7 +454,10 @@ function EnterAmount({
             description={t('sendEnterAmountScreen.notEnoughBalanceForGasWarning.description', {
               feeTokenSymbol: prepareTransactionsResult.feeCurrencies[0].symbol,
             })}
-            style={styles.warning}
+            style={[
+              styles.warning,
+              hideNetworkFee && { marginTop: Spacing.Regular16, marginBottom: 0 },
```

Comment:
This is needed to make spacing correct. Usually there is spacing below the network fee but when that is disabled, the top spacing needs to be here. Similarly, there shouldn't be bottom spacing in the earn case because the EarnProceed has top spacing to be in the correct spot below the enter amount component.